crates/ruff_python_parser/src/error.rs

```diff
@@ -449,6 +449,7 @@ pub enum UnsupportedSyntaxErrorKind {
     Match,
     Walrus,
     ExceptStar,
+    PosOnly,
```

Comment:
We try to avoid using abbreviations. 
```suggestion
    PositionalOnlyParameter,
```